cnd/src/network/comit_ln.rs

```diff
@@ -109,20 +121,29 @@ impl ComitLN {
     ) -> anyhow::Result<()> {
         let digest = create_swap_params.clone().digest();
 
-        if self.swaps_waiting_for_announcement.contains_key(&digest) {
+        if self.swaps_waiting_for_announcement.contains_key(&digest)
+            || self.swaps_waiting_for_confirmation.contains_key(&digest)
+        {
             anyhow::bail!(SwapExists)
         }
+        tracing::info!("Swap created: {}", digest);
```

Comment:
I think this log message is pretty misleading, we are not "creating" anything here.